### PR TITLE
Add gem «bullet»

### PR DESCRIPTION
**Pull request by dpaluy**
Watch SQL queries during development
https://github.com/flyerhzm/bullet

Set `ENV['BULLET'] = 'true'` to turn it on.

Best practise adding this as a part of CI

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,6 +74,7 @@ gem 'validates_lengths_from_database', '~> 0.7.0'
 gem 'ejs','~> 1.1'
 
 group :development, :test do
+  gem 'bullet'
```
Comment (yivo) [inline Gemfile]: @dpaluy please put version range for it, we also don't need it in `test`

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
Hi @dpaluy 

Thank you for the contribution. Could you please fix above issues? Thanks.

**Comment by dpaluy** [comment]
@yivo I included it in `test` env as well in order to include it as a part of your integration test in Jenkins.
You won't be able to test all cases manually.

File: config/environments/development.rb [hunk 2]
```diff
@@ -48,4 +48,11 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Bullet gem config
+  config.after_initialize do
+    Bullet.enable = (ENV['BULLET'] == 'true') ? true : false
```
Comment (yivo) [inline config/environments/development.rb]: @dpaluy Can you change this to `ENV['BULLET'].in?(%w[1 true])`. It seems more logically to me.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by dpaluy** [comment]
@yivo I updated per your request

**Comment by yivo** [comment]
@dpaluy Sorry but I meant to keep just `Bullet.enable = ENV['BULLET'].in?(%w[1 true])`. Why do you add `true`, `false` here? It makes no sense. Also duplication of configs makes no sense. I mind to include this kind of tools in test. They are already very slow and dirty.

**Comment by dpaluy** [comment]
@yivo My mistake on `ENV`.
The config is different between `development` and `test`.

The best practise is running a nightly build (slow tests) with Bullet to provide a report on query quality.
Other builds won't use this tool in test environment.

If you skip it from test env, you won't be able to track all problematic cases. Because, testing manually doesn't scale.

**Comment by yivo** [comment]
@dpaluy Thank you.

**Review by yivo — APPROVED** [review]

**Pull request merged**